tests/test_static.py

```diff
@@ -695,3 +695,46 @@ def test_set_range_error(client, patch_open):
     assert resp.status == falcon.HTTP_404
     assert patch_open.current_file is not None
     assert patch_open.current_file.closed
+
+
+def test_etag(client, patch_open):
+    mtime = 1736617934.133701
+    patch_open(mtime=mtime)
+
+    client.app.add_static_route('/assets/', '/opt/somesite/assets')
+    resp = client.simulate_request(path='/assets/css/main.css')
+    assert resp.headers['ETag'] == f'"{int(mtime):x}-{len(resp.text):x}"'
+
+
+def test_if_none_match(client, patch_open):
```

Comment:
Would it be possible to [parametrize](https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) this test? At first glance, the asserts and tests around `resp1`-`resp4` show some potential for DRYing.